tests/fsharp/tests.fs

```diff
@@ -1737,6 +1737,22 @@ module RegressionTests =
 
         peverify cfg "test.exe"
 #endif
+#if !FSHARP_SUITE_DRIVES_CORECLR_TESTS
+    [<Test>]
+    let ``5768`` () =
```

Comment:
I don't understand why these tests aren't a simple new single-file test under `typecheck\sigs\...` like all the other negative tests. You don't have to write all this diffing code manually.

Reply:
I wasn't aware of this! :)
Thanks for the pointer